src/boundary/Obstacle.h

```diff
@@ -54,28 +55,42 @@ class Obstacle {
     size_t getStrideY() { return m_j2 - m_j1 + 1; }
     size_t getStrideZ() { return m_k2 - m_k1 + 1; }
```

Comment:
could be renamed with _ and be marked const

Reply:
done somewhere